app/controllers/administrative-units/administrative-unit/ministers/new.js

```diff
@@ -14,9 +14,10 @@ export default class AdministrativeUnitsAdministrativeUnitMinistersNewController
   @service router;
   @service store;
 
-  queryParams = ['personId'];
+  queryParams = ['personId', 'positionId'];
```

Comment:
Same comment as for the file `administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/new.js`

Reply:
indeed, I ended up doing the initialization in the route